client/apps/edit/components/section_video/test/index.coffee

```diff
@@ -22,6 +22,7 @@ describe 'SectionVideo', ->
       props = {
```

Comment:
You'll need to require the RichTextComponent with icons to get this test running - - see a working example here: https://github.com/artsy/positron/blob/master/client/apps/edit/components/section_image/test/index.coffee#L29